Helper/Order.php

```diff
@@ -714,12 +733,35 @@ public function saveUpdateOrder($reference, $storeId = null, $boltTraceId = null
                     if ($transaction->status == Payment::TRANSACTION_AUTHORIZED) {
                         $this->voidTransactionOnBolt($transaction->id, $storeId);
                     }
-                    $this->bugsnag->notifyException($exception);
-                    return $this;
+
+                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                //// Allow missing quote failed hooks to resend 10 times so the Administrator can be notified via email
+                ///  On the eleventh time that the failed hook was sent to Magento, we return $this in order to have the hook return successfully.
+                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+                    if (!$this->featureSwitches->isLogMissingQuoteFailedHooksEnabled()) {
+                        return $this;
```

Comment:
We should keep the bugsnag notifyException here since it's in the old functionality, right?

Reply:
@matt-thomason I've updated the code. Can you check?